packages/_shared/src/utils/entityHelpers.test.ts

```diff
@@ -0,0 +1,197 @@
+import { getEntityStatus, type EntitySys } from './entityHelpers';
+
+describe('getEntityStatus', () => {
+  function createEntity(
+    props: Pick<
+      EntitySys,
+      'archivedVersion' | 'type' | 'deletedVersion' | 'publishedVersion' | 'fieldStatus' | 'version'
+    >
+  ) {
+    return props as EntitySys;
+  }
+
+  describe.each(['Entry', 'Asset'])('for entity type %s', (type) => {
+    describe('archived', () => {
+      test('returns archived if there is an archivedVersion', () => {
+        const result = getEntityStatus(
+          createEntity({
+            archivedVersion: 1,
+            type,
+            version: 1,
+          })
+        );
+
+        expect(result).toEqual('archived');
+      });
+    });
+
+    describe('deleted', () => {
+      test('returns deleted if there is an deletedVersion', () => {
+        const result = getEntityStatus(
+          createEntity({
+            deletedVersion: 1,
+            type,
+            version: 1,
+          })
+        );
+
+        expect(result).toEqual('deleted');
+      });
+    });
+
+    describe('publish status', () => {
+      describe('by field status', () => {
+        test('returns changed if at least one fieldStatus is changed', () => {
+          const result = getEntityStatus(
+            createEntity({
+              fieldStatus: {
+                '*': {
+                  'en-US': 'changed',
+                  'de-DE': 'published',
+                  'fr-FR': 'draft',
+                },
+              },
+              type,
+              version: 1,
+            }),
+            ['en-US', 'de-DE', 'fr-FR']
+          );
+
+          expect(result).toEqual('changed');
+        });
+
+        test('returns changed if at least one selected locale has a fieldStatus which is changed', () => {
```

Comment:
Isn't this the same as the one above it 🤔

Reply:
you're right, will remove it 👍